doc/manual-wiki/ppx.wiki

```diff
@@ -0,0 +1,83 @@
+= TyXML Ppx syntax extension =
+
+TyXML provides a ppx to allow writing of Html5 and Svg documents
+in a native syntax while still enjoying the benefits and type-safety of TyXML.
```

Comment:
to allow <del>of</del> writing <del>of</del> HTML and SVG documents _in the standard syntax_